container.go

```diff
@@ -239,41 +218,55 @@ func terminate(profile *Profile, all bool) error {
 	} else {
 		f.Add("label", "com.viam.canon.profile="+profile.name+"/"+profile.Arch)
 	}
-	containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{Filters: f})
+	containers, err := cli.ContainerList(ctx, container.ListOptions{All: true, Filters: f})
 	if err != nil {
 		return err
 	}
 	if len(containers) > 1 && !all {
 		return errors.New("multiple matching containers found, please retry with '--all' option")
 	}
 	for _, c := range containers {
-		fmt.Printf("terminating %s\n", c.Labels["com.viam.canon.profile"])
-		err = multierr.Combine(err, cli.ContainerStop(context.Background(), c.ID, container.StopOptions{}))
+		if terminate {
+			fmt.Printf("terminating %s\n", c.Labels["com.viam.canon.profile"])
+			err = errors.Join(err, cli.ContainerRemove(ctx, c.ID, container.RemoveOptions{Force: true}))
```

Comment:
Can you remove a non-stopped container? like this? I had expected something more like
```go
cli.ContainerStop(...)
if terminate {
    cli.ContainerRemove(...)
}
```
but I'm not good at docker, and maybe this is fine as-is.

Reply:
Force = true!
So yes, you can!

It SIGKILLS what's in the container though. Fine because we don't care about the state left over, but not something you'd want to do in cloud/production environments where you might have a containerized application that's in the middle of a transaction on a network and needs a moment to gracefully stop.